### ISSUE TITLE
Error with Fixed-effect coefficients (All NaN)

### ISSUE THREAD

**Issue by roninsharma** [weizhouUMICH/SAIGE#38]
Hello all,

I am running SAIGE v0.26.6 and all of the fixed-effect coefficients are showing up as NaN.  How can I fix this?

Here is the entire error:

Fixed-effect coefficients:
      [,1]
 [1,]  NaN
 [2,]  NaN
 [3,]  NaN
 [4,]  NaN
 [5,]  NaN
 [6,]  NaN
 [7,]  NaN
 [8,]  NaN
 [9,]  NaN
[10,]  NaN
[11,]  NaN
[12,]  NaN
[13,]  NaN
[14,]  NaN
[15,]  NaN
[16,]  NaN
[17,]  NaN
[18,]  NaN
[19,]  NaN
[20,]  NaN
[21,]  NaN
[22,]  NaN
[23,]  NaN
Error in if (max(abs(alpha - alpha0)/(abs(alpha) + abs(alpha0) + tol.coef)) <  :
  missing value where TRUE/FALSE needed
Calls: fitNULLGLMM ... system.time -> glmmkin.ai_PCG_Rcpp_Binary -> Get_Coef
Timing stopped at: 298 3.256 159.1
Execution halted
Running step 2
Error in parse_args(parser, positional_arguments = 0) :
  Error in getopt(spec = spec, opt = args) :
  " " is not a valid option, or does not support an argument
Execution halted

**Comment by weizhouUMICH** [comment]
Hi @roninsharma, 

Can you please try most recent versions of SAIGE? Sorry for taking so long to reply. Thanks!

Wei

**Comment by carbocation** [comment]
For what it's worth, I encountered a similar error with SAIGE version 0.36.3 on a binary trait. Will retry with the current version (0.36.6).

Full stderr output:
```
Loading required package: optparse
Error in if (max(abs(alpha - alpha0)/(abs(alpha) + abs(alpha0) + tol.coef)) <  : 
  missing value where TRUE/FALSE needed
Calls: fitNULLGLMM ... system.time -> glmmkin.ai_PCG_Rcpp_Binary -> Get_Coef_LOCO
Timing stopped at: 2.536e+06 2214 4.068e+04
Execution halted
```

**Comment by carbocation** [comment]
Following up - for me, this bug (which is present in 0.36.3) is fixed in 0.36.6. Thanks!
